recipes/DiffPriv/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "DiffPriv" %}
+{% set version = "1.0.0a0" %}
```

Comment:
This is an alpha release so it needs to be published according [the CFEP-05 rules](https://github.com/conda-forge/cfep/blob/master/cfep-05.md). Here are the docs on how to do it: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds